lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -19,8 +20,10 @@ const defaultItemsCountFormatter = (kind: string) => (n: number) => `${n} item${
     defaultAvailableItemsCountFormatter = defaultItemsCountFormatter('available'),
     defaultSelectedItemsCountFormatter = defaultItemsCountFormatter('transferred');
 
-export default function NxTransferList<T extends string | number>(props: Props<T>) {
+export default function NxTransferList
+<T extends string | number = string, P extends Set<T> | T[] = Set<T>>(props: Props<T, P>) {
```

Comment:
I think this is the first time we've had type parameters this long but the way I'd format this would probably be
```suggestion
export default function NxTransferList<
  T extends string | number = string,
  P extends Set<T> | T[] = Set<T>
>(props: Props<T, P>) {
```

Thoughts?

Reply:
> I think this is the first time we've had type parameters this long but the way I'd format this would probably be
> 
> Thoughts?

Thanks! Yeah I like it formatted this way as well!